hwc_display.ini

```diff
@@ -43,6 +44,15 @@ FLOAT_DISPLAY="0:320+180+1600+900"
 # cloned-physical-display-number: the display which should clone physical-display-number.
 CLONE_DISPLAY="1+2"
 
+# HWC reserved DRM Plane. 4 DRM planes are available.
+# 0 - Primary Plane
+# 1 - Overlay Plane
+# 2 - Overlay Plane
+# 3 - Cursor Plane
+# 0:0+1+2+3 - ALl plane on display 0 are reserved for HWC
```

Comment:
a little confusing. suggest not to use 'reserve' here. 'All 0/1/2/3 planes of display 0 are used for HWC'